doc/howtos/backend.rst

```diff
@@ -1004,9 +1004,13 @@ behavior:
 
     Values are Python expressions. For each record, the expression is evaluated
     with the record's attributes as context values and if ``true``, the
-    corresponding style is applied to the row. Other context values are
-    ``uid`` (the id of the current user) and ``current_date`` (the current date
-    as a string of the form ``yyyy-MM-dd``).
+    corresponding style is applied to the row. Here are some of the other values
+    available in the context:
+
+    * ``uid``: the id of the current user,
+    * ``today``: the current local date as a string of the form ``YYYY-MM-DD``,
+    * ``now``: UTC version of ``today`` with the addition of the current time.
```

Comment:
i'd remove the local/UTC information from here (x3)

Reply:
Done